plugins/status/plugin.go

```diff
@@ -295,32 +299,14 @@ func (p *Plugin) reconfigure(config interface{}) {
 	newConfig := config.(*Config)
 
 	if reflect.DeepEqual(p.config, *newConfig) {
-		p.logDebug("Status reporter configuration unchanged.")
+		p.logger.Debug("Status reporter configuration unchanged.")
 		return
 	}
 
-	p.logInfo("Status reporter configuration changed.")
+	p.logger.Info("Status reporter configuration changed.")
 	p.config = *newConfig
 }
 
-func (p *Plugin) logError(fmt string, a ...interface{}) {
-	logrus.WithFields(p.logrusFields()).Errorf(fmt, a...)
-}
-
-func (p *Plugin) logInfo(fmt string, a ...interface{}) {
-	logrus.WithFields(p.logrusFields()).Infof(fmt, a...)
-}
-
-func (p *Plugin) logDebug(fmt string, a ...interface{}) {
-	logrus.WithFields(p.logrusFields()).Debugf(fmt, a...)
-}
-
-func (p *Plugin) logrusFields() logrus.Fields {
-	return logrus.Fields{
-		"plugin": Name,
```

Comment:
🔍 Don't we lose this field...? I can't see it set. Ah wait, it's here, right? https://github.com/open-policy-agent/opa/pull/3204/files#diff-b79e7abe4775b140ab93ddf1cffec8b4f755b47fb2483257a5c74f8d937a66f3R198

I guess we could have the receiving party (the plugin) add its fields, too, but this way is OK as well.